tests/test_models.py

```diff
@@ -40,9 +40,23 @@ def test_sealed_instance_reverse_parent_link_access(self):
         with self.assertRaisesMessage(SealedObject, message):
             instance.sealion_ptr
 
+    def test_sealed_instance_reverse_related_attribute_access(self):
+        instance = Location.from_db('default', ['latitude', 'longitude'], [1.2, 3.4])
+        instance._state.sealed = True
+        message = "Cannot fetch many-to-many field visitors on a sealed object."
+        with self.assertRaisesMessage(SealedObject, message):
+            instance.visitors.all()
+
```

Comment:
This can be dropped, I included it in `test_sealed_instance_reverse_foreign_key_access` above.